### ISSUE TITLE
As a visitor, I want to directly see the latest message of the code

### ISSUE THREAD

**Issue by MxTheo** [MxTheo/travelingguestbook#27]
When you search for a code, you do not search for a code. You search for the message that is left for you. Therefore, after entering the code, you should be directed towards your message! In that page you can then click that you read the message, see #22 
Only after that you are directed towards the code detail page, where you can read more messages. There should also be the focus more on the messages.

**Comment by MxTheo** [comment]
This one can be closed and created a new issue #31 for the updating of the read by the visitor
